### PR TITLE
Navbar issue #6

### LINKED ISSUE #6: NavBar project list don't refresh after adding new project
When a new project is added to the database, the list of projects in the NavBar don't list it.

### PR DESCRIPTION
**Pull request by joaojesus81**
fixed the issue regarding the navbar not reloading the new project added. this created an emitter that will work for all router views. Fixes #6

### CODE REVIEW

File: src/composables/getProjects.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const getProjects = () => {
 
   const loadProjects = async () => {
     try {
+      console.log("ive reloaded");
```
Comment (jcvsilva) [inline src/composables/getProjects.js:9]: Please remove the console.log .

**Review by jcvsilva — CHANGES_REQUESTED** [review]
Please remove console.log

File: src/composables/getProjects.js (hunk 1 above)
Comment (joaojesus81) [reply src/composables/getProjects.js]: resolved

**Review by jcvsilva — APPROVED** [review]
Perfect.

**Comment by jcvsilva** [comment]
Closes #6

**Pull request merged**